docs/sources/datasources/azure-monitor/_index.md

```diff
@@ -201,6 +226,39 @@ For details on workload identity, refer to the [Azure workload identity document
    workload_identity_token_file = TOKEN_FILE_PATH
    ```
 
+### Configure Current User Authentication
+
+{{< admonition type="note" >}}
+This feature is experimental and subject to change or removal. Aspects of Grafana may not work as expected when using this authentication method.
+{{< /admonition >}}
+
+If your Grafana instance is configured with Azure Entra (formerly Active Directory) authentication for login, this authentication method can be used to forward the currently logged in user's credentials to the data source. The users credentials will then be used when requesting data from the data source. For details on how to configure your Grafana instance using Azure Entra refer to the [documentation][configure-grafana-azure-auth].
+
+This method of authentication does not inherently support backend functionality as a user's credentials will not be in scope e.g. alerting, reporting, recorded queries. In order to support backend queries when using a data source configured with current user authentication, it is possible to configure service credentials. Also, note that query and resource caching will be disabled by default for data sources using current user authentication.
+
+{{< admonition type="note" >}}
+To configure fallback service credentials the [feature toggle][configure-grafana-feature-toggles] `idForwarding` must be set to `true` and `user_identity_fallback_credentials_enabled` must be enabled in the [Azure configuration section][configure-grafana-azure] (enabled by default when `user_identity_enabled` is set to `true`).
+{{< /admonition >}}
+
+Permissions for fallback credentials may need to be broad in order to appropriately support backend functionality. For example, an alerting query created by a user will be dependent on their permissions. If a user creates an alert for a resource that the fallback credentials cannot access, the alert will fail.
```

Comment:
Have made this change but I've kept `the alert will fail` rather than `the creation fails` as the creation will succeed but the alert itself will error.